ui/src/integrations/ethereum/hooks/ethereum-hooks.tsx

```diff
@@ -147,6 +156,8 @@ export namespace EthereumHooks {
       children?: React.ReactNode | React.ReactNode[];
       config?: ProviderConfig<TFunctionName>;
     }
+
+    export type SettingsParam = { value?: string };

```

Comment:
Does this ned also the other params like gas price etc.?

Reply:
We might add it as they are needed. I searched for a type aggregator from ethers library with everything that may go with this settings object, but as I could not find I'have added just what we are using.